lib/themes/dosomething/paraneue_dosomething/templates/reportback/reportback-permalink.tpl.php

```diff
@@ -12,7 +11,8 @@
 
 <?php endif ?>
 
-<?php echo $rb_image ?>
+<?php echo $rb['image'] ?>
+<?php echo $rb['caption'] ?>
 
 <?php // Used to determine if the curernt logged in user is the user who submitted the rb. ?>
 <?php if ($is_current): ?>
```

Comment:
Totally minor, but maybe `$is_owner` makes this a little more readable (current could potentially mean the latest Reportback File submitted for the Reportback)